core/src/test/scala/unit/kafka/server/ClientQuotasRequestTest.scala

```diff
@@ -166,14 +168,11 @@ class ClientQuotasRequestTest extends BaseRequestTest {
 
   @Test
   def testClientQuotasForScramUsers(): Unit = {
-    val entityType = ConfigType.User
     val userName = "user"
 
-    val mechanism = ScramMechanism.SCRAM_SHA_256
-    val credential = new ScramFormatter(mechanism).generateCredential("password", 4096)
-    val configs = adminZkClient.fetchEntityConfig(entityType, userName)
-    configs.setProperty(mechanism.mechanismName, ScramCredentialUtils.credentialToString(credential))
-    adminZkClient.changeConfigs(entityType, userName, configs)
+    val results = createAdminClient().alterUserScramCredentials(util.Arrays.asList(
```

Comment:
You need to close the admin client after creating it.  I don't think Scala has try-with-resources, so you can just use a `try`... `finally` block.